netbox_onboarding/migrations/0004_onboardingtask_remove_created_on.py

```diff
@@ -0,0 +1,13 @@
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("netbox_onboarding", "0003_onboardingtask_change_logging_model"),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(name="onboardingtask", options={},),
+        migrations.RemoveField(model_name="onboardingtask", name="created_on",),
+    ]
```

Comment:
Ideally it would be good to squash these two migrations together.

Also, I'm not sure how difficult it would be, but ideally you should migrate existing `created_on` values to the new `created` field, as right now it looks like any existing OnboardingTask instances will get a new (or null?) `created` value when you migrate?

Reply:
> Per our discussion on Slack, by using `ChangeLoggedModel` we also automatically get `ObjectChange` records created when an `OnboardingTask` is created or updated, and `ObjectChange` includes a `user` field identifying which user was responsible for the creation/update of the `OnboardingTask`. That's really the primary driver for this PR as I understand it now - that was the context I was previously missing, thanks for the clarification.

Exactly, in earlier iterations we used to have a dedicated column for `owner` of the task and a bit over-modified view for getting and saving the username